Makefile

```diff
@@ -156,6 +150,7 @@ endif
 .PHONY: int-test
 int-test:
 ifdef CI
+	go install gotest.tools/gotestsum@v1.8.2 && \
```

Comment:
have you monitored the time difference with running `go install`?

Reply:
Seems to consistently take < 1s... I'm ok with this, since people have had problems forgetting to use `make deps` in the past.
```
% time go install github.com/goreleaser/goreleaser@v1.15.2
go install github.com/goreleaser/goreleaser@v1.15.2  0.89s user 0.61s system 248% cpu 0.603 total
% time go install github.com/goreleaser/goreleaser@v1.15.2
go install github.com/goreleaser/goreleaser@v1.15.2  0.89s user 0.61s system 265% cpu 0.562 total
% time go install github.com/goreleaser/goreleaser@v1.15.2
go install github.com/goreleaser/goreleaser@v1.15.2  0.89s user 0.63s system 270% cpu 0.565 total
```